packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -282,6 +273,60 @@ export default {
          * focus on the input field.
          */
 
+        describeFirstnameErrorMessage () {
+            if (this.$v.firstName.$dirty) {
+                if (!this.$v.firstName.required) {
+                    return 'Please include your first name';
```

Comment:
Should these come from a translation file? (or is that planned to move these out later)